src/view/views/SettingsView.tsx

```diff
@@ -98,6 +112,27 @@ const CameraViewSelect = () => {
   );
 };
 
+const PhysicsDebugSelect = () => {
+  const vm = useVM();
+  const [physicsDebug, setPhysicsDebug] = useState(vm.isDebugMode());
```

Comment:
I suppose you deliberately chose `useState` over `useSelector` so that you don't have to make this part of a slice?   The other settings options are implemented via `useSelector`.